Entities/Common/Building/BlockPlacement.as

```diff
@@ -173,7 +173,7 @@ void onRender(CSprite@ this)
 			if (bc.cursorClose && bc.hasReqs && bc.buildable)
 			{
 				SColor color;
-				Vec2f aimpos = bc.tileAimPos + getCamera().getInterpolationOffset();
+				Vec2f aimpos = bc.tileAimPos;
```

Comment:
Whoops that wasn't supposed to get there :P
Would be nice to get it really fixed and working with interpolation but that's still better than the bug we have...